index.html

```diff
@@ -376,10 +376,7 @@ <h3 id="w3c-coordination">W3C Groups</h3>
         <section>
             <h3 id="external-coordination">External Organizations</h3>
             <dl>
-                <dt><a href="https://datatracker.ietf.org/group/secdir/about/">Internet Engineering Task Force Security Area Directorate</a> </dt>
-                <dd>
-                    To coordinate broad horizontal reviews on the output of the Working Group among the security working groups at IETF.
-                </dd>
+                <dt><a href="https://datatracker.ietf.org/group/secdir/about/">Internet Engineering Task Force Security Area Advisory Group</a> </dt>
```

Comment:
The removed \<dd> is important. The SAAG won't do broad horizontal reviews, will it @samuelweiler? The reason the SAD was in there before was to tell us "Oh, you want X... then you need to talk with Y." Will the SAAG do that for us now (I'm not familiar w/ them)?

I guess the question is -- who is going to tell us "If you want that sort of review, you need to go /over there/."?